src/test/rpc/Transaction_test.cpp

```diff
@@ -694,15 +694,13 @@ class Transaction_test : public beast::unit_test::suite
     }
 
     void
-    testRequest(FeatureBitset features)
+    testRequest(FeatureBitset features, unsigned apiVersion)
```

Comment:
Also a nit: I know unsigned = unsigned int but I think it might be better to have the same everywhere aka` unsigned int`. But this is personal preference and I'm fine with both :)

Reply:
or better if we have an alias for `unsigned int` for apiVersions

Reply:
That kind of a thing is probably best left for `.clang-format` rules . As for strong typing - I am for it, but that's (similarly to formatting rules) is outside of the scope of this PR

Reply:
okay 👍